src/server/c-interop-native-extensions.md

```diff
@@ -1,24 +1,23 @@
 ---
 title: Native extensions for the standalone Dart VM
 description: The original way for command-line Dart apps to call C/C++ functions.
+toc: false
 ---
 
 {{site.alert.note}}
-The extension mechanism which was previously discussed on this page --
-native extensions -- has been deprecated in Dart 2.14 and will be removed in
+The extension mechanism that was previously discussed on this page —
+_native extensions_ — was deprecated in Dart 2.14 and will be removed in
 Dart 2.15.
 
-If you need to call existing code written in C or C++, see
-[C & C++ interop using FFI](/server/c-interop).
+If you need to call existing code written in C or C++, see the
+[FFI documentation](/server/c-interop).
 
-This deprecation does not affect use-cases where the Dart VM 
-is embedded as a library into another application. 
-The Dart Embedding API (as defined in the [`include/dart_api.h`][] header) 
-remains supported for such use cases. 
-The Dart team currently does not provide any guidelines
-on how to embed the Dart VM into your application, 
-however there are [examples][] maintained by the community
-which demonstrate the necessary glue code.
+One part of the extension mechanism —
```

Comment:
I am not sure I would call Embedding API "a part of the extension mechanism". The original phrasing tried to convey that "embedding" and "native extensions" are two different things even though they share some similarities and "embedding" is still supported.

Reply:
OK, I've tweaked the wording. ptal